### DISCUSSION TITLE
Bigquery support

### DISCUSSION THREAD

**Discussion by johnfelipe** [RamiAwar/dataline#346] [Ideas]
Even struct data compatibility?

**Comment by RamiAwar** [comment]
Huh? 😄 

We do plan on adding bigquery support. It's already kinda supported by the LangchainDB class we use, we just need to expose it.

**Reply by johnfelipe** [reply]
can u teach me how expose it?

**Reply by RamiAwar** [reply]
Do you know how to use Python + FastAPI + SQLAlchemy? Optionally React? 😅
